elastic/kibana
# [Fleet] Add support for fields of type aggregate_metric_double

## Summary

Add support for fields of type `aggregate_metric_double` in EPM.

Change in package spec introduced in https://github.com/elastic/package-spec/pull/500.

Fixes https://github.com/elastic/kibana/issues/154867
Closes https://github.com/elastic/package-spec/issues/457

### Checklist

Delete any items that are not applicable to this PR.


### How to test

* Modify a package to include a field with type `aggregate_metric_double`, for example like this:
```
    - name: some_metric
      type: aggregate_metric_double
      metrics: [ "min", "max", "sum", "value_count" ]
      default_metric: "max"
```
* Install elastic package with this branch of the package-spec included: https://github.com/elastic/package-spec/pull/500, for this, from an elastic-package working directory:
  * `go mod edit -replace github.com/elastic/package-spec/v2=github.com/elastic/package-spec@main`
  * `go mod tidy`
  * `make install`
* Build the package with `elastic-package build -v`.
* Install the package with `elastic-package install -v`.
* Check that the template contains the expected field.
